readthedocs/restapi/templates/restapi/footer.html

```diff
@@ -11,6 +15,7 @@
     <div class="rst-other-versions">
       {% endif %}
 
+      {% cache 60 footer_html_versions project.pk %}
```

Comment:
I think we should cache the sections "Versions" and "Languages" in different blocks. Adding/Removing a language won't invalidate the cache otherwise.

Also, why `footer_html_version`? Should be just `versions` the right variable to check? And, I think it should be `versions.count` at least

Reply:
Also, this wasn't caught on review of the front page, but whenever `cache` tag is used, we need to account for the user language as a separate argument as well. Without this, the language of the first user to cache this will be used.